### DISCUSSION TITLE
"Start Afresh"

### DISCUSSION THREAD

**Discussion by ChrisAWC** [axolotl-chat/axolotl#378] [Q&A]
Greetings Community,
I'm looking for some help.
I would like to "start afresh" with axolotl. I have installed and used axolotl ok, but one photo that I tried to send has "got stuck" (see Bug Report #330).
Uninstalling the app is straight forward, but how do I clear the "data" or "the history"? If I don't do this, when I reinstall the app, the problem of the unsent photo is unresolved.
Many thanks for any help
Regards
ChrisAWC

**Comment by Flaburgan** [comment]
Sorry I didn't take time to have a look at your bug report. I will try to do it this week. If you really want to wipe Axolotl data, you can do so using the app UTTT that you can find in the OpenStore. You are using Ubuntu Touch right?

**Comment by ChrisAWC** [comment]
Hi, And many thanks for your reply. Yes, I am using Ubuntu Touch.
I have used UTTT to delete the data, and uninstalled, then reinstalled Axolotl.
Sadly, I am now unable to add a contact. 
I will post a bug report. (#381)
Regards
